### DISCUSSION TITLE
how to remake build if error build

### DISCUSSION THREAD

**Discussion by devzhr** [fastlane/fastlane#17381] [Q&A]
### how to remake build if error build ?

`error: Build input file cannot be found: '/Users/ci/Documents/App/AppTarget/R.generated.swift' (in target 'AppTarget' from project 'App')`

**Comment by rogerluan** [comment]
Hey @devzhr 👋 
We need more context to your problem. Sounds like fixing the build would be a first step so that it doesn't fail again. This should trigger a new CI build which should proceed with building your app just fine. Am I missing something?
